pkg/apis/athenz/v1/types.go

```diff
@@ -60,6 +60,7 @@ func (in *AthenzDomainSpec) DeepCopy() *AthenzDomainSpec {
 // AthenzDomainStatus stores status information about the current resource
 type AthenzDomainStatus struct {
 	Message string `json:"message,omitempty"`
+	zms.JWSDomain
```

Comment:
Can we add ``json:",inline"`` here?

Reply:
Makes sense, I was just trying to replicate what you had for `AthenzDomainSpec` struct type. This is done - https://github.com/yahoo/k8s-athenz-syncer/pull/31/commits/c82251d87cf38f9e543770fbe186276f903ad6ab.